test/plugin/test_watch_pods.rb

```diff
@@ -24,140 +24,141 @@ class DefaultPodWatchStrategyTest < WatchTest
      include KubernetesMetadata::WatchPods
 
      setup do
-       @initial = Kubeclient::Common::EntityList.new(
-         'PodList',
-         '123',
-         [
-           Kubeclient::Resource.new({
-                                      'metadata' => {
-                                        'name' => 'initial',
-                                        'namespace' => 'initial_ns',
-                                        'uid' => 'initial_uid',
-                                        'labels' => {},
-                                      },
-                                      'spec' => {
-                                        'nodeName' => 'aNodeName',
-                                        'containers' => [{
-                                                           'name' => 'foo',
-                                                           'image' => 'bar',
-                                                         }, {
-                                                           'name' => 'bar',
-                                                           'image' => 'foo',
-                                                         }]
-                                      }
-                                    }),
-           Kubeclient::Resource.new({
-                                      'metadata' => {
-                                        'name' => 'modified',
-                                        'namespace' => 'create',
-                                        'uid' => 'modified_uid',
-                                        'labels' => {},
-                                      },
-                                      'spec' => {
-                                        'nodeName' => 'aNodeName',
-                                        'containers' => [{
-                                                           'name' => 'foo',
-                                                           'image' => 'bar',
-                                                         }, {
-                                                           'name' => 'bar',
-                                                           'image' => 'foo',
-                                                         }]
-                                      }
-                                    }),
-         ])
-       @created = OpenStruct.new(
+       @initial = {
+         kind: 'PodList',
+         metadata: {resourceVersion: '123'}, # TODO: resourceVersion is always blank
```

Comment:
What do you mean? In test?  In real life?

Reply:
fixed, was a leftover comment from debugging